src/main/java/seedu/address/logic/commands/ListContactedInfoCommand.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.ArrayList;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.contactedinfo.ContactedInfo;
+import seedu.address.model.person.Person;
+
+/**
+ * Lists all reminders for a contact.
+ */
+public class ListContactedInfoCommand extends Command {
+    public static final String COMMAND_WORD = "listcontacted";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Lists all contacted information for a specified contact. "
+            + "Parameters: "
+            + "INDEX\n"
+            + "Example: " + COMMAND_WORD + " "
+            + "3\n";
+
+    public static final String MESSAGE_SUCCESS = "Listed all contacted information for %1$s:\n%2$s";
+
+    private final Index index;
+
+    /**
+     * Creates a ListContactRemindersCommand to list the reminders.
+     */
+    public ListContactedInfoCommand(Index index) {
+        this.index = index;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
```

Comment:
Check if valid person, for instance, if the index does not have a person.  One based index <=0 or >= filtered list.size()